alternator/conditions.cc

```diff
@@ -163,6 +164,14 @@ static bool check_IN(const rjson::value* val, const rjson::value& array) {
     return have_match;
 }
 
+// Check if a JSON-encoded value is null.
```

Comment:
I think this comment is wrong. check_NULL does not check for a "JSON-encoded value is null" - rather it checks whether the attributes exists or not.